fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/DefaultPropertiesLoader.java

```diff
@@ -90,16 +102,21 @@ public void loadSystemProperties() {
             for (final PROPERTIES prop : PROPERTIES.values()) {
                 final String value = getProperty(prop.getValue());
                 if (value == null) {
-                    setProperty(prop.getValue(), baseDir);
+                    if (prop.getSetDefaultValue()) {
+                        setProperty(prop.getValue(), baseDir);
+                    }
                 } else {
                     updateRelativePropertyPath(prop.getValue(), value, baseDir);
                 }
             }
         }
 
         for (final PROPERTIES prop : PROPERTIES.values()) {
-            final String val = prop.getValue();
-            LOGGER.info("{} = {}", val, getProperty(val));
+            final String name = prop.getValue();
+            final String val = getProperty(name);
+            if (val != null) {
```

Comment:
I would suggest that it is actually helpful to print the `what-ever-property = null` in the case that a property has not been set. It may help show an administrator that a property was not set... when they thought it was.

Reply:
Okay, it should now display
`INFO 08:57:27.709 (DefaultPropertiesLoader) fcrepo.external.content.allowed = null`